packages/react-select/src/Select.js

```diff
@@ -608,7 +723,9 @@ export default class Select extends Component<Props, State> {
     });
     this.announceAriaLiveContext({
       event: 'menu',
-      context: { isDisabled: isOptionDisabled(options[nextFocus]) },
+      context: {
+        isDisabled: this.isOptionDisabled(options[nextFocus], selectValue),
+      },
```

Comment:
I'm assuming this is a bug (introduced [here](https://github.com/JedWatson/react-select/pull/3445/files#diff-7064e1eb0f7889ae735e705d019ebfa160fbb668d7733f73ee6f107b8cf16fadR591)). Instead of calling the builtin `isOptionDisabled`, we should be calling `this.isOptionDisabled` (that everything else calls) which calls `this.props.isOptionDisabled`.